packages/terra-application/src/application-navigation/private/common/_PopupMenuUser.jsx

```diff
@@ -32,6 +43,17 @@ const PopupMenuUser = ({ userConfig }) => {
       <div className={cx('info-container')}>
         <div aria-hidden className={cx('name')}>{userConfig.name}</div>
         {userConfig.detail ? <div className={cx('detail')}>{userConfig.detail}</div> : null}
+        { userActionConfig && userActionConfig.text && userActionConfig.userActionCallback && (
```

Comment:
Since the `text` and `userActionCallback` props for userActionConfigPropType are required, is it still necessary to check for their existence?

Reply:
If user forgets to add any of the two props, without the check button will be displayed. So to prevent the button from being displayed I added all 3 in the check.
Screenshot without text prop and without check.
<img width="201" alt="Screenshot 2022-08-26 at 11 07 15 PM" src="https://user-images.githubusercontent.com/42207428/186961257-fde1afbf-4ffc-4e16-bd53-a224133d806f.png">

Reply:
Ok. These [props](https://github.com/cerner/terra-application/pull/305/files#diff-7d87712268f49c48e74296b6d16d5adfe16c61d64035195f4584b68253a559bfR102-R108) are marked as required so I expect that they should always be provided.

Reply:
Done [374d274](https://github.com/cerner/terra-application/pull/305/commits/374d2742dd8d6324aedbb27942023a5584700f77)